src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/configuration/BundleConfiguration.java

```diff
@@ -52,6 +52,6 @@ public BundleConfiguration(@JsonProperty("title") String title,
         this.documents = documents == null ? new ArrayList<>() : documents;
         this.paginationStyle = paginationStyle;
         this.documentNameValue = documentNameValue;
-        this.enableEmailNotification = enableEmailNotification;
+        this.enableEmailNotification = enableEmailNotification != null ? enableEmailNotification : null;
```

Comment:
that's weird.... why is that necessary?

Reply:
Sorry, looked back at this now and you're right. It is unnecessary.